_migrations/getting-started-data-migration.md

```diff
@@ -0,0 +1,331 @@
+* The source cluster must be deployed with the S3 plugin.
+* The target cluster must be deployed.
+
+Using this guide assumes the following:
+
+* A snapshot will be taken and stored in S3 in this guide, and the following assumptions are made about this snapshot:
+  * The `_source` flag is enabled on all indexes that will be migrated.
+  * The snapshot includes the global cluster state (`include_global_state` is `true`).
+  * Shard sizes up to approximately 80 GB are supported. Larger shards can not be migrated. If this is blocker for your migration, contact the [migration's team](https://opensearch.slack.com/archives/C054JQ6UJFK).
+* Migration Assistant will be installed in the same region and have access to both the source snapshot and target cluster.
+
+---
+
+## Step 1: Installing bootstrap on an AWS EC2 instance (~10 mins)
+
+To begin your migration, use the following steps to install a `bootstrap` box on an AWS EC2 instance. The instance uses CloudFormation to create and manage the stack.
+
+1. Log into the target AWS account where you want to deploy the Migration Assistant.
+2. From the browser where you are logged into your target AWS account, right-click [here](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?templateURL=https://solutions-reference.s3.amazonaws.com/migration-assistant-for-amazon-opensearch-service/latest/migration-assistant-for-amazon-opensearch-service.template&redirectId=SolutionWeb) to load the CloudFormation  template from a new browser tab.
+3. Follow the CloudFormation stack wizard:
+   * **Stack Name:** `MigrationBootstrap`
+   * **Stage Name:** `dev`
+   * Hit **Next** on each step, acknowledge on the fourth screen, and hit **Submit**.
+4. Verify that the bootstrap stack exists and is set to `CREATE_COMPLETE`. This process takes around 10 minutes.
+
+---
+
+## Step 2: Setting up bootstrap instance access (~5 mins)
+
+Use the following steps to set up bootstrap instance access:
+
+1. After deployment, find the EC2 instance ID for the `bootstrap-dev-instance`.
+2. Create an IAM policy using the following snippet, replacing `<aws-region>`, `<aws-account>`, `<stage>`, and `<ec2-instance-id>` with your information:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": "ssm:StartSession",
+                "Resource": [
+                    "arn:aws:ec2:<aws-region>:<aws-account>:instance/<ec2-instance-id>",
+                    "arn:aws:ssm:<aws-region>:<aws-account>:document/BootstrapShellDoc-<stage>-<aws-region>"
+                ]
+            }
+        ]
+    }
+    ```
+
```

Comment:
Should there be a {% include copy-curl.html %} here?

Reply:
Yes. I'll add those to the code examples here.